config/rbac/resourcedistribution_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit resourcedistributions.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: resourcedistribution-editor-role
+rules:
+- apiGroups:
+  - apps.kruise.io
+  resources:
+  - resourcedistributions
+  verbs:
+  - create
+  - delete
```

Comment:
is create/delete operation required ? i think create/delete is initiated by paas or users, not kruise manager